### PR TITLE
[Owners] Update digital pattern and tclk system tests to use device_server

### PR DESCRIPTION
**Pull request by doshirohan**
### What does this Pull Request accomplish?

DeviceServer was introduced for system tests in [PR 196](https://github.com/ni/grpc-device/pull/196). This PR updates digital pattern and tclk system tests to use centralized device server.

### Why should this Pull Request be merged?

System tests follow common pattern used by other system tests for lifetime management of session sever.

### What testing has been done?

- Digital pattern and tclk system tests pass

### CODE REVIEW

File: source/tests/system/nitclk_driver_api_tests.cpp [hunk 1]
```diff
@@ -1,8 +1,7 @@
 #include <gtest/gtest.h>
 
-#include "niscope/niscope_library.h"
+#include "device_server.h"
```
Comment (kruthi-NI) [inline source/tests/system/nitclk_driver_api_tests.cpp:3]: @doshirohan , I could not find changes for tclk session tests

Comment (doshirohan) [reply source/tests/system/nitclk_driver_api_tests.cpp:3]: We do not have session tests for tclk.

Comment (kruthi-NI) [reply source/tests/system/nitclk_driver_api_tests.cpp:3]: Oh right. Sorry had forgot

**Review by kruthi-NI — APPROVED** [review]

File: source/tests/system/nidigital_session_tests.cpp [hunk 2]
```diff
@@ -58,12 +47,8 @@ class NiDigitalSessionTest : public ::testing::Test {
   }
 
  private:
-  std::shared_ptr<::grpc::Channel> channel_;
-  std::unique_ptr<digital::NiDigital::Stub> nidigital_stub_;
-  std::unique_ptr<nidevice_grpc::SessionRepository> session_repository_;
-  std::unique_ptr<digital::NiDigitalLibrary> nidigital_library_;
-  std::unique_ptr<digital::NiDigitalService> nidigital_service_;
-  std::unique_ptr<::grpc::Server> server_;
+   DeviceServerInterface* device_server_;
```
Comment (jonlugoatx) [inline source/tests/system/nidigital_session_tests.cpp:50]: Indentation from 3 --> 2 spaces

**Review by jonlugoatx — APPROVED** [review]

File: source/tests/system/nidigital_session_tests.cpp (hunk 2 above)
Comment (doshirohan) [reply source/tests/system/nidigital_session_tests.cpp:50]: Fixed indentation.

**Pull request merged**